lib/vue

```diff
@@ -1 +1 @@
-Subproject commit 79f289a2f22871a667b695687e35e22ef0fe2002
+Subproject commit 783ecc9bdf4fab6613e6526cccbca01d816dd275
```

Comment:
Thanks for the PR, this commit is really old, can you `git submodule update` to the latest one?